test/extensions/filters/network/http_connection_manager/config_filter_chain_test.cc

```diff
@@ -0,0 +1,285 @@
+#include "extensions/filters/network/http_connection_manager/config.h"
+
+#include "test/extensions/filters/network/http_connection_manager/config_test_base.h"
+#include "test/mocks/config/mocks.h"
+#include "test/mocks/http/mocks.h"
+#include "test/mocks/network/mocks.h"
+#include "test/mocks/server/factory_context.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::_;
+using testing::Eq;
+using testing::Return;
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace HttpConnectionManager {
```

Comment:
Nit: tests should live in the anonymous namespace in here.

Reply:
Done.